cartographer_rviz/cartographer_rviz/drawable_submap.h

```diff
@@ -95,11 +101,14 @@ class DrawableSubmap : public QObject {
   const int submap_index_;
 
   ::cartographer::common::Mutex mutex_;
+  ::rviz::DisplayContext* const context_;
   Ogre::SceneManager* const scene_manager_;
   Ogre::SceneNode* const scene_node_;
+  Ogre::SceneNode* const submap_node_;
```

Comment:
I see that the submap_node_ is made a child node of the scene_node_, but why is the scene_node_ not enough?